core/src/type_info/simple_bson_type_info.rs

```diff
@@ -0,0 +1,453 @@
+use crate::{
+    definitions::SqlDataType,
+    type_info::{TypeInfoFields, SQL_PRED_BASIC, SQL_PRED_NONE, SQL_SEARCHABLE},
+};
+
+#[non_exhaustive]
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct SimpleBsonTypeInfo {
+    pub type_info_fields: TypeInfoFields,
+}
+
+impl SimpleBsonTypeInfo {
```

Comment:
Almost all of the constants defined below are the same as `StandardBsonTypeInfo` with the only change being that the `sql_type` and `non_concise_type` is now `VARCHAR` in places where it was `UNKNOWN_TYPE`. Also, I changed `is_case_sensitive` to `true` for the types I changed to `VARCHAR`. However, some of them had more changes. These ones are `OBJECTID`, `NULL`, `UNDEFINED`, `TIMESTAMP`, `MINKEY`, `MAXKEY`,